qiskit_experiments/curve_analysis/fit_function.py

```diff
@@ -77,6 +77,17 @@ def gaussian(
     return amp * np.exp(-((x - x0) ** 2) / (2 * sigma ** 2)) + baseline
 
 
+def lorentzian(
```

Comment:
Should this be something like `sqrt_lorentzian` to save confusion if someone later on wants a real lorentzian function?

Reply:
Yes. See here: fac4995